src/main/kotlin/org/rust/cargo/CargoProjectDescription.kt

```diff
@@ -1,119 +1,84 @@
 package org.rust.cargo
 
-import com.intellij.util.PathUtil
-import org.rust.cargo.commands.impl.CargoMetadata
-import java.io.File
-import java.io.Serializable
-import java.util.*
+import com.intellij.openapi.vfs.*
+import com.intellij.util.containers.BidirectionalMap
+import com.intellij.util.containers.MultiMap
 
+
+/**
+ * Rust project model represented roughly in the same way as in Cargo itself.
+ *
+ * [org.rust.cargo.toolchain.CargoMetadataService] is responsible for providing a [CargoProjectDescription] for
+ * an IDEA module.
+ */
 class CargoProjectDescription private constructor(
-    val packages: Collection<Package>,
-    private val rootModule: Package
+    private val idToPackage: BidirectionalMap<Int, Package>,
```

Comment:
What are the reasons to submit `idToPackage`? I can only spot just a single usage: `idToPackage.values`.